### PR TITLE
Store data model

### PR DESCRIPTION
**Pull request by phlax**
- Add data model for Store
- Add tools for efficiently retrieving data

### CODE REVIEW

File: tests/models/datatables.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        == '<StoreData: %s>' % store0.pootle_path)
+
+
+@pytest.mark.django_db
+def test_data_tp_bad():
+    """Test that you cant add a duplicate file extension
+    """
+    # needs a TP
+    with pytest.raises(IntegrityError):
+        StoreData.objects.create()
```
Comment (unho) [inline tests/models/datatables.py]: Which is the difference between this test and `test_data_store_bad` above?

Comment (phlax) [inline tests/models/datatables.py]: good spot - it should test tp not store

**Comment by unho** [comment]
Nit on tests aside, looks gtm. But I don't see it being used anywhere.

**Comment by phlax** [comment]
@unho its not used in this PR - i separated out because i felt that the data model was worth reviewing by itself - the next PR adds some tools for getting/setting this data and event handlers to update as necessary

**Pull request merged**